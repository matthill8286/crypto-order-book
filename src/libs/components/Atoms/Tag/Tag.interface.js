
//# sourceMappingURL=Tag.interface.js.map