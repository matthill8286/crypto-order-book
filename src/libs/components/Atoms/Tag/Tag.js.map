{"version":3,"sources":["../../../../src/components/Atoms/Tag/Tag.tsx"],"names":["React","Typo","StyledAnchorTag","StyledTagWrapper","TagText","color","children","weight","Tag","text","href","targetBlank","marginBottom","marginRight","padding","className","undefined"],"mappings":"AAAA,OAAO,KAAKA,KAAZ,MAAuB,OAAvB;AACA,SAASC,IAAT;AAEA,SAASC,eAAT,EAA0BC,gBAA1B;;AAEA,IAAMC,OAA+B,GAAG,SAAlCA,OAAkC,OAA8C;AAAA,MAA3CC,KAA2C,QAA3CA,KAA2C;AAAA,MAApCC,QAAoC,QAApCA,QAAoC;AAAA,MAA1BC,MAA0B,QAA1BA,MAA0B;AACpF,SACE,oBAAC,IAAD;AAAM,IAAA,MAAM,EAAEA,MAAd;AAAsB,IAAA,KAAK,EAAE,CAACF,KAAD,GAAS,OAAT,GAAmBA,KAAhD;AAAuD,IAAA,QAAQ,EAAE,IAAjE;AAAuE,IAAA,UAAU,EAAC;AAAlF,KACGC,QADH,CADF;AAKD,CAND;;AAQA,OAAO,IAAME,GAAuB,GAAG,SAA1BA,GAA0B,QAUpB;AAAA,MATjBC,IASiB,SATjBA,IASiB;AAAA,MARjBC,IAQiB,SARjBA,IAQiB;AAAA,MAPjBC,WAOiB,SAPjBA,WAOiB;AAAA,MANjBC,YAMiB,SANjBA,YAMiB;AAAA,MALjBC,WAKiB,SALjBA,WAKiB;AAAA,MAJjBR,KAIiB,SAJjBA,KAIiB;AAAA,MAHjBE,MAGiB,SAHjBA,MAGiB;AAAA,MAFjBO,OAEiB,SAFjBA,OAEiB;AAAA,MADjBC,SACiB,SADjBA,SACiB;AACjB,SACE,oBAAC,gBAAD;AACE,IAAA,SAAS,EAAEA,SADb;AAEE,IAAA,YAAY,EAAEH,YAFhB;AAGE,IAAA,WAAW,EAAEC,WAHf;AAIE,IAAA,OAAO,EAAE,CAAC,CAACH,IAJb;AAKE,IAAA,OAAO,EAAEI,OALX;AAME,IAAA,KAAK,EAAET;AANT,KAOGK,IAAI,GACH,oBAAC,eAAD;AAAiB,IAAA,IAAI,EAAEA,IAAvB;AAA6B,IAAA,MAAM,EAAEC,WAAW,GAAG,QAAH,GAAcK;AAA9D,KACE,oBAAC,OAAD;AAAS,IAAA,MAAM,EAAET,MAAjB;AAAyB,IAAA,KAAK,EAAEF;AAAhC,KACGI,IADH,CADF,CADG,GAOH,oBAAC,OAAD;AAAS,IAAA,KAAK,EAAEJ;AAAhB,KAAwBI,IAAxB,CAdJ,CADF;AAmBD,CA9BM","sourcesContent":["import * as React from 'react'\nimport { Typo } from '../Typography'\nimport { TagProps, TagTextProps } from './Tag.interface'\nimport { StyledAnchorTag, StyledTagWrapper } from './Tag.styled'\n\nconst TagText: React.FC<TagTextProps> = ({ color, children, weight }): JSX.Element => {\n  return (\n    <Typo weight={weight} color={!color ? 'grey3' : color} fontSize={'xs'} lineHeight=\"xxs\">\n      {children}\n    </Typo>\n  )\n}\n\nexport const Tag: React.FC<TagProps> = ({\n  text,\n  href,\n  targetBlank,\n  marginBottom,\n  marginRight,\n  color,\n  weight,\n  padding,\n  className,\n}): JSX.Element => {\n  return (\n    <StyledTagWrapper\n      className={className}\n      marginBottom={marginBottom}\n      marginRight={marginRight}\n      hasLink={!!href}\n      padding={padding}\n      color={color}>\n      {href ? (\n        <StyledAnchorTag href={href} target={targetBlank ? '_blank' : undefined}>\n          <TagText weight={weight} color={color}>\n            {text}\n          </TagText>\n        </StyledAnchorTag>\n      ) : (\n        <TagText color={color}>{text}</TagText>\n      )}\n    </StyledTagWrapper>\n  )\n}\n"],"file":"Tag.js"}