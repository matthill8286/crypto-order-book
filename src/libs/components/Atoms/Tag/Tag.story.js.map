{"version":3,"sources":["../../../../src/components/Atoms/Tag/Tag.story.tsx"],"names":["number","select","storiesOf","React","saiyanTheme","loremIpsumArray","StorybookWrapper","Tag","Readme","module","add","left","right","info","top","bottom","selectedColor","Object","keys","color","tags","slice","range","min","max","length","step","map","t","display","flexWrap"],"mappings":"AAAA,SAASA,MAAT,EAAiBC,MAAjB,QAA+B,wBAA/B;AACA,SAASC,SAAT,QAA0B,kBAA1B;AACA,OAAO,KAAKC,KAAZ,MAAuB,OAAvB;AACA,SAASC,WAAT;AAEA,SAASC,eAAT;AACA,SAASC,gBAAT;AACA,SAASC,GAAT;AACA,OAAOC,MAAP;AAEAN,SAAS,CAAC,yBAAD,EAA4BO,MAA5B,CAAT,CACGC,GADH,CAEI,SAFJ,EAGI,YAAM;AACJ,SAAO,oBAAC,GAAD;AAAK,IAAA,IAAI,EAAC,WAAV;AAAsB,IAAA,OAAO,EAAE;AAAEC,MAAAA,IAAI,EAAE,IAAR;AAAcC,MAAAA,KAAK,EAAE;AAArB;AAA/B,IAAP;AACD,CALL,EAMI;AACEC,EAAAA,IAAI,EAAEL;AADR,CANJ,EAUGE,GAVH,CAWI,gBAXJ,EAYI,YAAM;AACJ,SACE,oBAAC,GAAD;AAAK,IAAA,IAAI,EAAC,UAAV;AAAqB,IAAA,OAAO,EAAE;AAAEC,MAAAA,IAAI,EAAE,IAAR;AAAcC,MAAAA,KAAK,EAAE,IAArB;AAA2BE,MAAAA,GAAG,EAAE,KAAhC;AAAuCC,MAAAA,MAAM,EAAE;AAA/C;AAA9B,IADF;AAGD,CAhBL,EAiBI;AACEF,EAAAA,IAAI,EAAEL;AADR,CAjBJ,EAqBGE,GArBH,CAsBI,SAtBJ,EAuBI,YAAM;AACJ,MAAMM,aAAa,GAAGf,MAAM,CAAC,OAAD,EAAUgB,MAAM,CAACC,IAAP,CAAYd,WAAW,CAACe,KAAxB,CAAV,EAA0C,OAA1C,CAA5B;AAEA,SACE,oBAAC,GAAD;AACE,IAAA,IAAI,EAAC,4BADP;AAEE,IAAA,IAAI,EAAC,GAFP;AAGE,IAAA,WAAW,EAAE,IAHf;AAIE,IAAA,KAAK,EAAEH,aAJT;AAKE,IAAA,OAAO,EAAE;AAAEL,MAAAA,IAAI,EAAE,IAAR;AAAcC,MAAAA,KAAK,EAAE;AAArB;AALX,IADF;AASD,CAnCL,EAoCI;AACEC,EAAAA,IAAI,EAAEL;AADR,CApCJ,EAwCGE,GAxCH,CAyCI,MAzCJ,EA0CI,YAAM;AACJ,SACE,oBAAC,GAAD;AACE,IAAA,IAAI,EAAC,4BADP;AAEE,IAAA,IAAI,EAAC,GAFP;AAGE,IAAA,WAAW,EAAE,IAHf;AAIE,IAAA,OAAO,EAAE;AAAEC,MAAAA,IAAI,EAAE,IAAR;AAAcC,MAAAA,KAAK,EAAE;AAArB;AAJX,IADF;AAQD,CAnDL,EAoDI;AACEC,EAAAA,IAAI,EAAEL;AADR,CApDJ,EAwDGE,GAxDH,CAyDI,WAzDJ,EA0DI,YAAM;AACJ,MAAMU,IAAI,GAAGf,eAAe,CACzBgB,KADU,CAET,CAFS,EAGTrB,MAAM,CAAC,gBAAD,EAAmB,EAAnB,EAAuB;AAC3BsB,IAAAA,KAAK,EAAE,IADoB;AAE3BC,IAAAA,GAAG,EAAE,CAFsB;AAG3BC,IAAAA,GAAG,EAAEnB,eAAe,CAACoB,MAHM;AAI3BC,IAAAA,IAAI,EAAE;AAJqB,GAAvB,CAHG,EAUVC,GAVU,CAUN,UAAAC,CAAC;AAAA,WAAI,oBAAC,GAAD;AAAK,MAAA,GAAG,EAAEA,CAAV;AAAa,MAAA,IAAI,EAAEA,CAAnB;AAAsB,MAAA,WAAW,MAAjC;AAAkC,MAAA,YAAY,MAA9C;AAA+C,MAAA,WAAW;AAA1D,MAAJ;AAAA,GAVK,CAAb;AAYA,SACE,oBAAC,gBAAD;AAAkB,IAAA,KAAK,EAAE;AAAEC,MAAAA,OAAO,EAAE,MAAX;AAAmBC,MAAAA,QAAQ,EAAE;AAA7B;AAAzB,KAAiEV,IAAjE,CADF;AAGD,CA1EL,EA2EI;AACEP,EAAAA,IAAI,EAAEL;AADR,CA3EJ","sourcesContent":["import { number, select } from '@storybook/addon-knobs'\nimport { storiesOf } from '@storybook/react'\nimport * as React from 'react'\nimport { saiyanTheme } from '@/styles/sc-vars-saiyan'\nimport { ThemeColors } from '@/types/theme'\nimport { loremIpsumArray } from '@/utils/loremIpsumArray'\nimport { StorybookWrapper } from '@/utils/StorybookWrapper'\nimport { Tag } from './Tag'\nimport Readme from './Tag.readme.md'\n\nstoriesOf('Design System/Atoms/Tag', module)\n  .add(\n    'Default',\n    () => {\n      return <Tag text=\"Some text\" padding={{ left: 'lg', right: 'lg' }} />\n    },\n    {\n      info: Readme,\n    }\n  )\n  .add(\n    'Custom Padding',\n    () => {\n      return (\n        <Tag text=\"Featured\" padding={{ left: 'md', right: 'md', top: 'xxs', bottom: 'xxs' }} />\n      )\n    },\n    {\n      info: Readme,\n    }\n  )\n  .add(\n    'Colored',\n    () => {\n      const selectedColor = select('Color', Object.keys(saiyanTheme.color), 'black') as ThemeColors\n\n      return (\n        <Tag\n          text=\"some text with anchor link\"\n          href=\"#\"\n          targetBlank={true}\n          color={selectedColor}\n          padding={{ left: 'sm', right: 'sm' }}\n        />\n      )\n    },\n    {\n      info: Readme,\n    }\n  )\n  .add(\n    'Link',\n    () => {\n      return (\n        <Tag\n          text=\"some text with anchor link\"\n          href=\"#\"\n          targetBlank={true}\n          padding={{ left: 'sm', right: 'sm' }}\n        />\n      )\n    },\n    {\n      info: Readme,\n    }\n  )\n  .add(\n    'Many Tags',\n    () => {\n      const tags = loremIpsumArray\n        .slice(\n          0,\n          number('number of tags', 20, {\n            range: true,\n            min: 0,\n            max: loremIpsumArray.length,\n            step: 1,\n          })\n        )\n        .map(t => <Tag key={t} text={t} targetBlank marginBottom marginRight />)\n\n      return (\n        <StorybookWrapper style={{ display: 'flex', flexWrap: 'wrap' }}>{tags}</StorybookWrapper>\n      )\n    },\n    {\n      info: Readme,\n    }\n  )\n"],"file":"Tag.story.js"}