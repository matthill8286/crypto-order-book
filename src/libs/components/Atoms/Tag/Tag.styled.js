import { css, styled } from "../../../styles/styled";
import { handlePadding } from "../Card/Card.styled";
export var StyledTagWrapper = styled.div.withConfig({
  displayName: "Tagstyled__StyledTagWrapper",
  componentId: "k5mbhe-0"
})(function (_ref) {
  var theme = _ref.theme,
      hasLink = _ref.hasLink,
      marginRight = _ref.marginRight,
      color = _ref.color,
      padding = _ref.padding;
  return css(["color:", ";", ";border-color:", ";border-width:1px;border-radius:", ";border-style:solid;width:fit-content;margin-right:", ";margin-bottom:", ";", ";"], theme.color.grey5, !!padding && handlePadding(theme.spacing.base, padding), !color ? theme.color.grey2 : theme.color[color], theme.dimension.borderRadius6, marginRight && theme.spacing.base.sm, marginRight && theme.spacing.base.xs, hasLink && hoverStyle(theme));
});

var hoverStyle = function hoverStyle(theme) {
  return css(["&:hover{background-color:", ";cursor:pointer;}"], theme.color.grey1);
};

export var StyledAnchorTag = styled.a.withConfig({
  displayName: "Tagstyled__StyledAnchorTag",
  componentId: "k5mbhe-1"
})(["text-decoration:none;color:", ";"], function (_ref2) {
  var theme = _ref2.theme;
  return theme.color.grey5;
});
//# sourceMappingURL=Tag.styled.js.map