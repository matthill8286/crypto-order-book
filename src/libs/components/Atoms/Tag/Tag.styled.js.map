{"version":3,"sources":["../../../../src/components/Atoms/Tag/Tag.styled.tsx"],"names":["css","styled","handlePadding","StyledTagWrapper","div","theme","hasLink","marginRight","color","padding","grey5","spacing","base","grey2","dimension","borderRadius6","sm","xs","hoverStyle","grey1","StyledAnchorTag","a"],"mappings":"AAAA,SAASA,GAAT,EAAcC,MAAd;AAGA,SAASC,aAAT;AAEA,OAAO,IAAMC,gBAAgB,GAAGF,MAAM,CAACG,GAAV;AAAA;AAAA;AAAA,GAC3B;AAAA,MAAGC,KAAH,QAAGA,KAAH;AAAA,MAAUC,OAAV,QAAUA,OAAV;AAAA,MAAmBC,WAAnB,QAAmBA,WAAnB;AAAA,MAAgCC,KAAhC,QAAgCA,KAAhC;AAAA,MAAuCC,OAAvC,QAAuCA,OAAvC;AAAA,SAAqDT,GAArD,4JACWK,KAAK,CAACG,KAAN,CAAYE,KADvB,EAEI,CAAC,CAACD,OAAF,IAAaP,aAAa,CAACG,KAAK,CAACM,OAAN,CAAcC,IAAf,EAAqBH,OAArB,CAF9B,EAGkB,CAACD,KAAD,GAASH,KAAK,CAACG,KAAN,CAAYK,KAArB,GAA6BR,KAAK,CAACG,KAAN,CAAYA,KAAZ,CAH/C,EAKmBH,KAAK,CAACS,SAAN,CAAgBC,aALnC,EAQkBR,WAAW,IAAIF,KAAK,CAACM,OAAN,CAAcC,IAAd,CAAmBI,EARpD,EASmBT,WAAW,IAAIF,KAAK,CAACM,OAAN,CAAcC,IAAd,CAAmBK,EATrD,EAUIX,OAAO,IAAIY,UAAU,CAACb,KAAD,CAVzB;AAAA,CAD2B,CAAtB;;AAcP,IAAMa,UAAU,GAAG,SAAbA,UAAa,CAACb,KAAD;AAAA,SAAkBL,GAAlB,qDAEKK,KAAK,CAACG,KAAN,CAAYW,KAFjB;AAAA,CAAnB;;AAOA,OAAO,IAAMC,eAAe,GAAGnB,MAAM,CAACoB,CAAV;AAAA;AAAA;AAAA,yCAEjB;AAAA,MAAGhB,KAAH,SAAGA,KAAH;AAAA,SAAeA,KAAK,CAACG,KAAN,CAAYE,KAA3B;AAAA,CAFiB,CAArB","sourcesContent":["import { css, styled } from '@/styles/styled'\nimport { Padding, Theme } from '@/types'\nimport { StyledTagWrapperProps } from './Tag.interface'\nimport { handlePadding } from '@/components/Atoms/Card/Card.styled'\n\nexport const StyledTagWrapper = styled.div<StyledTagWrapperProps>(\n  ({ theme, hasLink, marginRight, color, padding }) => css`\n    color: ${theme.color.grey5};\n    ${!!padding && handlePadding(theme.spacing.base, padding as Padding)};\n    border-color: ${!color ? theme.color.grey2 : theme.color[color]};\n    border-width: 1px;\n    border-radius: ${theme.dimension.borderRadius6};\n    border-style: solid;\n    width: fit-content;\n    margin-right: ${marginRight && theme.spacing.base.sm};\n    margin-bottom: ${marginRight && theme.spacing.base.xs};\n    ${hasLink && hoverStyle(theme)};\n  `\n)\nconst hoverStyle = (theme: Theme) => css`\n  &:hover {\n    background-color: ${theme.color.grey1};\n    cursor: pointer;\n  }\n`\n\nexport const StyledAnchorTag = styled.a`\n  text-decoration: none;\n  color: ${({ theme }) => theme.color.grey5};\n`\n"],"file":"Tag.styled.js"}