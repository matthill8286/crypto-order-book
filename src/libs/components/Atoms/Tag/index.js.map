{"version":3,"sources":["../../../../src/components/Atoms/Tag/index.ts"],"names":[],"mappings":"AAAA","sourcesContent":["export * from './Tag'\n"],"file":"index.js"}