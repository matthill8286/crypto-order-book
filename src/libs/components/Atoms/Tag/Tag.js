import * as React from 'react';
import { Typo } from "../Typography";
import { StyledAnchorTag, StyledTagWrapper } from "./Tag.styled";

var TagText = function TagText(_ref) {
  var color = _ref.color,
      children = _ref.children,
      weight = _ref.weight;
  return React.createElement(Typo, {
    weight: weight,
    color: !color ? 'grey3' : color,
    fontSize: 'xs',
    lineHeight: "xxs"
  }, children);
};

export var Tag = function Tag(_ref2) {
  var text = _ref2.text,
      href = _ref2.href,
      targetBlank = _ref2.targetBlank,
      marginBottom = _ref2.marginBottom,
      marginRight = _ref2.marginRight,
      color = _ref2.color,
      weight = _ref2.weight,
      padding = _ref2.padding,
      className = _ref2.className;
  return React.createElement(StyledTagWrapper, {
    className: className,
    marginBottom: marginBottom,
    marginRight: marginRight,
    hasLink: !!href,
    padding: padding,
    color: color
  }, href ? React.createElement(StyledAnchorTag, {
    href: href,
    target: targetBlank ? '_blank' : undefined
  }, React.createElement(TagText, {
    weight: weight,
    color: color
  }, text)) : React.createElement(TagText, {
    color: color
  }, text));
};
//# sourceMappingURL=Tag.js.map