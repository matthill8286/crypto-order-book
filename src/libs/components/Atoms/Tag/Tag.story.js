import { number, select } from '@storybook/addon-knobs';
import { storiesOf } from '@storybook/react';
import * as React from 'react';
import { saiyanTheme } from "../../../styles/sc-vars-saiyan";
import { loremIpsumArray } from "../../../utils/loremIpsumArray";
import { StorybookWrapper } from "../../../utils/StorybookWrapper";
import { Tag } from "./Tag";
import Readme from "./Tag.readme.md";
storiesOf('Design System/Atoms/Tag', module).add('Default', function () {
  return React.createElement(Tag, {
    text: "Some text",
    padding: {
      left: 'lg',
      right: 'lg'
    }
  });
}, {
  info: Readme
}).add('Custom Padding', function () {
  return React.createElement(Tag, {
    text: "Featured",
    padding: {
      left: 'md',
      right: 'md',
      top: 'xxs',
      bottom: 'xxs'
    }
  });
}, {
  info: Readme
}).add('Colored', function () {
  var selectedColor = select('Color', Object.keys(saiyanTheme.color), 'black');
  return React.createElement(Tag, {
    text: "some text with anchor link",
    href: "#",
    targetBlank: true,
    color: selectedColor,
    padding: {
      left: 'sm',
      right: 'sm'
    }
  });
}, {
  info: Readme
}).add('Link', function () {
  return React.createElement(Tag, {
    text: "some text with anchor link",
    href: "#",
    targetBlank: true,
    padding: {
      left: 'sm',
      right: 'sm'
    }
  });
}, {
  info: Readme
}).add('Many Tags', function () {
  var tags = loremIpsumArray.slice(0, number('number of tags', 20, {
    range: true,
    min: 0,
    max: loremIpsumArray.length,
    step: 1
  })).map(function (t) {
    return React.createElement(Tag, {
      key: t,
      text: t,
      targetBlank: true,
      marginBottom: true,
      marginRight: true
    });
  });
  return React.createElement(StorybookWrapper, {
    style: {
      display: 'flex',
      flexWrap: 'wrap'
    }
  }, tags);
}, {
  info: Readme
});
//# sourceMappingURL=Tag.story.js.map