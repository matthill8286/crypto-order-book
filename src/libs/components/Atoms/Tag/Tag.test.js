import React from 'react';
import { renderWithTheme } from "../../../testRenderer";
import { loremIpsumArray } from "../../../utils/loremIpsumArray";
import { Tag } from "./Tag";
describe('Tag', function () {
  var tagContent = 'Tag ';
  it('Should render a tag', function () {
    var tree = renderWithTheme(React.createElement(Tag, {
      text: ""
    }, tagContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a tag with link', function () {
    var tree = renderWithTheme(React.createElement(Tag, {
      text: "",
      href: "#",
      targetBlank: true
    }, tagContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render 20 tags', function () {
    var tags = loremIpsumArray.slice(0, 20).map(function (t) {
      return React.createElement(Tag, {
        key: t,
        text: t,
        targetBlank: true,
        marginBottom: true,
        marginRight: true
      });
    });
    var tree = renderWithTheme(React.createElement(React.Fragment, null, tags));
    expect(tree).toMatchSnapshot();
  });
});
//# sourceMappingURL=Tag.test.js.map