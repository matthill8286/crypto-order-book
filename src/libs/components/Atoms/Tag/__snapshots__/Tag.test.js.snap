// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag Should render 20 tags 1`] = `
Array [
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      lorem
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      ipsum
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      dolor
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      sit
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      amet
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      consectetur
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      adipiscing
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      elit
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      curabitur
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      vel
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      hendrerit
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      libero
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      eleifend
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      blandit
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      nunc
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      ornare
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      odio
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      ut
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      orci
    </span>
  </div>,
  <div
    className="Tagstyled__StyledTagWrapper-k5mbhe-0 jOgjTq"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      gravida
    </span>
  </div>,
]
`;

exports[`Tag Should render a tag 1`] = `
<div
  className="Tagstyled__StyledTagWrapper-k5mbhe-0 eDKiaB"
>
  <span
    className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
    color="grey3"
    fontSize="xs"
  >
    
  </span>
</div>
`;

exports[`Tag Should render a tag with link 1`] = `
<div
  className="Tagstyled__StyledTagWrapper-k5mbhe-0 cAcWYe"
>
  <a
    className="Tagstyled__StyledAnchorTag-k5mbhe-1 fZyvSX"
    href="#"
    target="_blank"
  >
    <span
      className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 iKQzKe"
      color="grey3"
      fontSize="xs"
    >
      
    </span>
  </a>
</div>
`;
